### ISSUE TITLE
mailnagapplet@ozderya.net : last version still doesn't work with cinnamon 4.8.0

### ISSUE THREAD

**Issue by Dany-Dan** [linuxmint/cinnamon-spices-applets#3352]
```
 * Applet version/Build date : 2020-12-03 14:01:52
 * Cinnamon version : 4.8.0
 * Distribution : Arch Linux
 ```

**Notify author of applet**
@hyOzd

**Issue**

Same problem as the previous applet version. The applet doesn't work with cinnamon 4.8.0.
Same error message : 

`error t=2020-12-07T15:12:15Z`
`[mailnagapplet@ozderya.net]: missing ( after for`
`[mailnagapplet@ozderya.net]: Error importing applet.js from mailnagapplet@ozderya.net`

**Steps to reproduce**

- update mailnag applet
- update to cinnamon 4.8.0
- restart cinnamon

**Comment by NikoKrause** [comment]
Could you try to download and install it manually from the website
https://cinnamon-spices.linuxmint.com/applets/view/244

**Comment by Dany-Dan** [comment]
This is what i have done so far :

- applet didn't work with cinnamon 4.8.0
- removed mailnag and mailnag applet
- updated cinnamon to new 4.8.1 version
- installed mailnag and mailnag applet (from cinnamon applets menu)
- applet not working either with cinnamon 4.8.1
- removed applet
- applet downloaded from the website and installed manually
- applet is working fine !

And then, out of curiosity :

- removed applet
- installed applet (from cinnamon applets menu)
- applet is now... also working fine

I don't understand where the problem was, but now the applet is working.
Problem solved for me.
Thank you !

**Comment by NikoKrause** [comment]
There's an issue in Cinnamon 4.6. The spices website caches applets.
Even if the spice shows to be updated, there's a chance, that it downloaded an older version of the applet.
This bug will be fixed in Cinnamon 4.8: https://github.com/linuxmint/cinnamon/commit/dd22f4e3d3da02c552981684c95dd7f326df14dd

**Comment by Dany-Dan** [comment]
Unfortunately, the applet starts and show up in the systray but it is still broken.
I had to fix the syntax in the code by myself to make it work.
